aiida_vasp/parsers/base.py

```diff
@@ -104,7 +103,7 @@ def _check_folder(self):
             return self.exit_codes.ERROR_NO_RETRIEVED_FOLDER
 
     @contextmanager
-    def _get_handler(self, name):
+    def _get_handler(self, name, open_mode):
```

Comment:
Probably people expect `_get_handler(name, mode='r')`. The argument name is so expected `mode` but not `open_mode`.

https://github.com/aiidateam/aiida-core/blob/80d11258075c1a251c7d2bd348e46957d4766267/aiida/orm/nodes/repository.py#L165